portal-web/test/functional/com/liferay/portalweb/macros/DMFolder.macro

```diff
@@ -251,6 +251,18 @@ definition {
 		AssertElementNotPresent(locator1 = "DocumentsAndMedia#DESCRIPTIVE_LIST_FOLDER_ICON");
 	}
 
+	macro downloadViaEllipsisMenu {
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
```

Comment:
This isn't needed in the macro. If you need to refresh the UI before clicking on the folder ellipsis you can use the Refresh function at the test case level to do it